src/main/resources/l10n/JabRef_en.properties

```
@@ -1782,7 +1782,7 @@ No\ full\ text\ document\ found\ for\ entry\ %0.=No full text document found for
 Delete\ Entry=Delete Entry
 Next\ library=Next library
 Previous\ library=Previous library
-add\ group=add group
+add\ group=Add group
```

Translation string must be consistent with english translation.
```suggestion
Add\ group=Add group
```

---

fixed